src/content/docs/new-relic-partnerships/partnerships/partner-api/partnership-api-account-object.mdx

```diff
@@ -186,7 +186,7 @@ Here are the Partnership API account object's attributes.
 
     An array defining a list of users. There are some restrictions in place dependent on the status of your account:
 
-    * For [customer partnerships](/docs/new-relic-partnerships/partner-integration-guide/partner-account-maintenance/partnership-accounts-users-subscriptions/) who have switched to the [New Relic One user model](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-user-models): do not define users. If you add users with this field, they will be ignored. Instead, to add users, you'd use [these user management procedures](/docs/accounts/accounts-billing/new-relic-one-user-management/introduction-managing-users/).
+    * For [customer partnerships](/docs/new-relic-partnerships/partner-integration-guide/partner-account-maintenance/partnership-accounts-users-subscriptions/) who have migrated their users to [our newer user model](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-user-models): do not define users. If you add users with this field, they will be ignored. Instead, to add users, you'd use [these user management procedures](/docs/accounts/accounts-billing/new-relic-one-user-management/introduction-managing-users/).
```

Comment:
@zuluecho9 you could capitalize "Do" to be consistent with the same construction below: "Do not define users."